_posts/2021-01-14-audio-mixing-terms.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title:  "Introduction to Mixing Audio Terms - What do they mean?"
```

Comment:
Rather that have one article for Audio terms, let's have a "Streaming Terms and Glossary" article that we link to. We could have sections in that article for "Audio", "Video", and "PC", then bucket the appropriate terms in those sections.

Reply:
maybe we should add it to a second page on the menu up to like a glossary or something next to the about page or something.